### PR TITLE
Add routing name group

### PR DESCRIPTION
**Pull request by RyanThomas31**
Add routing name group and updating ajax requests

### CODE REVIEW

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -197,16 +193,21 @@ function postAdd(post) {
...
 
             function postUpdate(post) {
+                var name = post.Name,
+                    cell = post.Cell,
+                    post = post.Post,
+                    id = $("#postid").val();
+
                 $.ajax({
-                    url: "http://127.0.0.1:8000/api/update/",
+                    url: "http://127.0.0.1:8000/api/posts/update/" + id + "?name=" + name + "&post=" + post + "&cell=" + cell,
```
Comment (emuroiwaa) [inline resources/views/welcome.blade.php]: API url should be an environment variable

File: routes/api.php [hunk 2]
```diff
@@ -19,17 +19,19 @@
     return $request->user();
 });
 
-//Get all Posts
-Route::get('posts', [PostsController::class,'getPosts']);
+Route::group(['prefix'=>'posts'], function(){
```
Comment (emuroiwaa) [inline routes/api.php]: add necessary white spaces

File: routes/api.php [hunk 3]
```diff
@@ -19,17 +19,19 @@
...
-Route::put('update/{id}', [PostsController::class,'updatePost']);
+    //Update Post
+    Route::put('/update/{id}', [PostsController::class,'updatePost']);
 
-//Delete Post
-Route::delete('delete/{id}',[PostsController::class,'deletePost']);
\ No newline at end of file
+    //Delete Post
+    Route::delete('/delete/{id}',[PostsController::class,'deletePost']);
+});
```
Comment (emuroiwaa) [inline routes/api.php]: no newline at EOF

File: resources/views/welcome.blade.php [hunk 4]
```diff
@@ -181,12 +173,16 @@ function updateClick() {
                     postAdd(Post);
                 } else {
                     postUpdate(Post);
-                }
-            }
+                };
+            };
 
             function postAdd(post) {
+                var name = post.Name,
```
Comment (emuroiwaa) [inline resources/views/welcome.blade.php]: ideally variable names and object key should be camel case or  https://www.codementor.io/@veenitchauhan/basics-of-naming-conventions-for-php-developers-eliexmew6

**Review by emuroiwaa — CHANGES_REQUESTED** [review]

File: resources/views/welcome.blade.php (hunk 4 above)
Comment (RyanThomas31) [reply resources/views/welcome.blade.php]: @emuroiwaa the URL above is talking about PHP variables, in this case on line 180 I'm making use of Javascript variables.

Comment (emuroiwa) [reply resources/views/welcome.blade.php]: Yes correct, do you have something similar for JS

Comment (RyanThomas31) [reply resources/views/welcome.blade.php]: so, in PHP you get the $variable naming convention which could be camel cased like $postName however in Javascript you declare a variable using var, let, const. I'm changing it to camel casing and adding more detail to describe what the variable is getting. Comitting my changes now

File: resources/views/welcome.blade.php [hunk 5]
```diff
@@ -177,12 +177,12 @@ function updateClick() {
...
-                var name = post.Name,
-                    cell = post.Cell,
-                    post = post.Post;
+                var userName = post.Name,
+                    postCell = post.Cell,
+                    postMessage = post.Post;
 
                 $.ajax({
-                    url: "{{ env('APP_URL')}}/api/posts/create?" + "name=" + name + "&post=" + post + "&cell=" + cell,
+                    url: "{{ env('APP_URL')}}/api/posts/create?" + "name=" + userName + "&post=" + postMessage + "&cell=" + postCell,
```
Comment (emuroiwaa) [inline resources/views/welcome.blade.php]: make spacing constant

File: resources/views/welcome.blade.php [hunk 6]
```diff
@@ -201,13 +201,13 @@ function postAddSuccess(post) {
...
-                    cell = post.Cell,
-                    post = post.Post,
+                var userName = post.Name,
+                    postCell = post.Cell,
+                    postMessage = post.Post,
                     id = $("#postid").val();
 
                 $.ajax({
-                    url: "{{ env('APP_URL')}}/api/posts/update/" + id + "?name=" + name + "&post=" + post + "&cell=" + cell,
+                    url: "{{ env('APP_URL')}}/api/posts/update/" + id + "?name=" + userName + "&post=" + postMessage + "&cell=" + postCell,
```
Comment (emuroiwaa) [inline resources/views/welcome.blade.php]: same here

**Review by emuroiwaa — CHANGES_REQUESTED** [review]

**Pull request merged**